src/checkin/services/flow/step/GetExtraPassengerInformationStep.ts

```diff
@@ -0,0 +1,42 @@
+import { Builder } from 'builder-pattern'
+import { Context } from '../../../model/Context'
+import { Session } from '../../../model/session'
+import { UserInformation } from '../../../model/session/UserInformation'
+import StepTemplate from '../StepTemplate'
+import { SessionData } from '../../../model/session/SessionData'
+
+export default class GetExtraPassengerInformationStep extends StepTemplate {
+
+  when(context: Context): boolean {
+    const numberOfPassengers = context.getSession().data.passengers
+    return numberOfPassengers > 1
```

Comment:
Can we add a condition on when also.
To not execute when extraPassengerInformation is already in session?